metricbeat/docs/modules/elasticsearch/ingest.asciidoc

```diff
@@ -0,0 +1,29 @@
+////
+This file is generated! See scripts/mage/docs_collector.go
+////
+:edit_url: https://github.com/elastic/beats/edit/main/metricbeat/module/elasticsearch/ingest/_meta/docs.asciidoc
+
+
+[[metricbeat-metricset-elasticsearch-ingest]]
+=== Elasticsearch ingest metricset
+
+beta[]
+
+include::../../../module/elasticsearch/ingest/_meta/docs.asciidoc[]
+
+This is a default metricset. If the host module is unconfigured, this metricset is enabled by default.
```

Comment:
host module?

Reply:
Looks like this was from the metricset generator, it was present in some of the other docs for this module too so I fixed those as well.